w_common/test/unit/browser/cache/cache_test.dart

```diff
@@ -579,9 +582,22 @@ void main() {
 
 class MockCachingStrategy extends Mock
     implements CachingStrategy<String, Object> {
-  MockCachingStrategy() {
-    when(onDidGet(any, any)).thenAnswer((i) => Future.value(null));
-    when(onDidRelease(any, any, any)).thenAnswer((i) => Future.value(null));
-    when(onDidRemove(any, any)).thenAnswer((i) => Future.value(null));
+  @override
+  Future<Null> onDidGet(String id, Object value) async {
```

Comment:
Rather than defining these, we could use the generator. https://github.com/Workiva/w_common/blob/934e88599339cc209183366b47a9709298239df2/w_common/build.yaml